### DISCUSSION TITLE
What does Auto Refresh do?

### DISCUSSION THREAD

**Discussion by Nairou** [nkanaev/yarr#220] [General]
I have Auto Refresh set to 30m, but I noticed after an hour that the page hadn't changed. Reloading the page showed new feed results.

Does the Auto Refresh only refer to the background fetch process? Or is it supposed to be refreshing the browser page as well?

I have yarr pinned to my browser, so it is always open, in hopes of seeing the title change when new feeds get downloaded. I can probably use a browser extension to periodically refresh the tab, but wanted to understand what yarr's Auto Refresh was supposed to be doing.

**Comment by nkanaev** [comment]
It currently refreshes on the background only. The fact that the frontend doesn't update with the new feeds is an oversight and probably needs to be resolved.

**Comment by dhewg** [comment]
This patch makes the frontend update periodically (5m)
[0001-refresh-the-data-periodically.patch.txt](https://github.com/user-attachments/files/18546300/0001-refresh-the-data-periodically.patch.txt)

Using websockets to make the backend notify the frontend instead would be more elegant, but I don't know if yarr wants to go there
